src/app/components/mno-apps/mno-app.html

```diff
@@ -109,10 +109,10 @@ <h2 class="media-heading">{{vm.app.name}}</h2>
       <div class="col-md-12">
         <div class="well">
           <h2>{{ 'mno_enterprise.templates.dashboard.marketplace.show.price' | translate }}</h2>
-          <div translate ng-if="!vm.arePlansAvailable">
+          <div translate ng-if="!vm.orderPossible">
```

Comment:
This should be `ng-if="!vm.pricing_plans"`

If there are no plans in your currency, but the order is still possible, we still want the "No plans available in your currency" message to appear. Otherwise there is just an empty pricing section.